asset/CMakeLists.txt

```diff
@@ -5,6 +5,15 @@ if(APPLE)
 	SET(ASSETS ${ASSETS}
 		${CMAKE_CURRENT_SOURCE_DIR}/brd.icns
 	)
+elseif(UNIX)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ofbv-app.svg
+		DESTINATION share/icons/hicolor/scalable/apps RENAME openboardview.svg)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.appdata.xml
+		DESTINATION share/appdata)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.desktop
+		DESTINATION share/applications)
+	install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/openboardview.xml
+		DESTINATION share/mime/packages)
```

Comment:
Agreed. Do you have a PR for srakitnican/OpenBoardView@8e5aaac (and any other useful changes)?

Probably should have a separate PR with that change and to refactor `build.sh` slightly to make use of DESTDIR